mpicbg/src/main/java/mpicbg/models/InterpolatedAffineModel3D.java

```diff
@@ -71,6 +70,13 @@ public void set( final InterpolatedAffineModel3D< A, B > m )
 			affine.set( ( ( InterpolatedAffineModel3D< A, B > ) m ).affine );
```

Comment:
Both, the type test and the type cast seem unnecessary.  Please remove and replace with
```java
affine.set( m.affine );
```